### PR TITLE
add missing tokens to tokens.erc20 and prices.usd

### PR DESCRIPTION
**Pull request by viniabussafi**
### Contribution type
Please check the type of contribution this pull request is for:

- [ ] New spell(s)
- [x] Adding to existing spell lineage
- [ ] Bug fix
- [ ] 
### For adding to existing spell lineage
If you are adding to an existing spell lineage, please provide the following information:

- **Description:** This PR adds tokens that were registered on balancer pools and are missing on tokens.erc20 and prices.usd. Sources of information have been added as comments to each token added.
Note: tests are going to fail due to zkevm not being properly set up yet

### CODE REVIEW

File: models/prices/arbitrum/prices_arbitrum_tokens.sql [hunk 1]
```diff
@@ -198,5 +198,6 @@ FROM
     ('a51-a51-finance','arbitrum','A51',0xb3f13b0c61d65d67d7d6215d70c89533ee567a91, 18),
     ('oseth-stakewise-staked-eth','arbitrum','osETH',0xf7d4e7273e5015c96728a6b02f31c505ee184603, 18),
     ('sliz-solidlizard-dex-token','arbitrum','SLIZ',0x463913D3a3D3D291667D53B8325c598Eb88D3B0e, 18),
-    ('vrtx-vertex-protocol', 'arbitrum', 'VRTX', 0x95146881b86b3ee99e63705ec87afe29fcc044d9, 18)
+    ('vrtx-vertex-protocol', 'arbitrum', 'VRTX', 0x95146881b86b3ee99e63705ec87afe29fcc044d9, 18),
+    ('gho-gho', 'arbitrum', 'GHO', 0x7dff72693f6a4149b17e7c6314655f6a9f7c8b33, 18),
```
Comment (viniabussafi) [inline models/prices/arbitrum/prices_arbitrum_tokens.sql:202]: https://coinpaprika.com/coin/gho-gho/

File: tokens/models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 2]
```diff
@@ -82,4 +82,7 @@ FROM (VALUES
     , (0xD460f305B150Bc70b063E806815B50BebE9F21b1, 'APRTo', 18)
     , (0x577fd586c9e6ba7f2e85e025d5824dbe19896656, 'SYNO', 18)
     , (0x6985884c4392d348587b19cb9eaaf157f13271cd, 'ZRO', 18)
+    , (0x5a7a183b6b44dc4ec2e3d2ef43f98c5152b1d76d, 'inETH', 18)
```
Comment (viniabussafi) [inline tokens/models/tokens/arbitrum/tokens_arbitrum_erc20.sql:85]: https://arbiscan.io/address/0x5a7a183b6b44dc4ec2e3d2ef43f98c5152b1d76d

File: tokens/models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 3]
```diff
@@ -82,4 +82,7 @@ FROM (VALUES
...
+    , (0xd08c3f25862077056cb1b710937576af899a4959, 'instETH', 18)
```
Comment (viniabussafi) [inline tokens/models/tokens/arbitrum/tokens_arbitrum_erc20.sql:86]: https://arbiscan.io/address/0xd08c3f25862077056cb1b710937576af899a4959

File: tokens/models/tokens/arbitrum/tokens_arbitrum_erc20.sql [hunk 4]
```diff
@@ -82,4 +82,7 @@ FROM (VALUES
...
+    , (0x7dff72693f6a4149b17e7c6314655f6a9f7c8b33, 'GHO', 18)
```
Comment (viniabussafi) [inline tokens/models/tokens/arbitrum/tokens_arbitrum_erc20.sql:87]: https://arbiscan.io/address/0x7dff72693f6a4149b17e7c6314655f6a9f7c8b33

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 5]
```diff
@@ -4566,4 +4566,8 @@ FROM (VALUES
     , ( 0x10dea67478c5f8c5e2d90e5e9b26dbe60c54d800, 'TAIKO', 18)
     , ( 0x80ac24aA929eaF5013f6436cdA2a7ba190f5Cc0b, 'syrupUSDC', 6)    
     , ( 0xa8258deE2a677874a48F5320670A869D74f0cbC1, 'WAR', 18)    
+    , ( 0xbeef69ac7870777598a04b2bd4771c71212e6abc, 'steakLRT', 18)
```
Comment (viniabussafi) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:4569]: https://etherscan.io/address/0xbeef69ac7870777598a04b2bd4771c71212e6abc

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 6]
```diff
@@ -4566,4 +4566,8 @@ FROM (VALUES
     , ( 0x10dea67478c5f8c5e2d90e5e9b26dbe60c54d800, 'TAIKO', 18)
     , ( 0x80ac24aA929eaF5013f6436cdA2a7ba190f5Cc0b, 'syrupUSDC', 6)    
     , ( 0xa8258deE2a677874a48F5320670A869D74f0cbC1, 'WAR', 18)    
+    , ( 0xbeef69ac7870777598a04b2bd4771c71212e6abc, 'steakLRT', 18)
+    , ( 0x84631c0d0081fde56deb72f6de77abbbf6a9f93a, 'Re7LRT', 18)
```
Comment (viniabussafi) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:4570]: https://etherscan.io/address/0x84631c0d0081fde56deb72f6de77abbbf6a9f93a

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 7]
```diff
@@ -4566,4 +4566,8 @@ FROM (VALUES
     , ( 0x10dea67478c5f8c5e2d90e5e9b26dbe60c54d800, 'TAIKO', 18)
     , ( 0x80ac24aA929eaF5013f6436cdA2a7ba190f5Cc0b, 'syrupUSDC', 6)    
     , ( 0xa8258deE2a677874a48F5320670A869D74f0cbC1, 'WAR', 18)    
+    , ( 0xbeef69ac7870777598a04b2bd4771c71212e6abc, 'steakLRT', 18)
+    , ( 0x84631c0d0081fde56deb72f6de77abbbf6a9f93a, 'Re7LRT', 18)
+    , ( 0x7a4effd87c2f3c55ca251080b1343b605f327e3a, 'rstETH', 18)
```
Comment (viniabussafi) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:4571]: https://etherscan.io/address/0x7a4effd87c2f3c55ca251080b1343b605f327e3a

File: tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql [hunk 8]
```diff
@@ -4566,4 +4566,8 @@ FROM (VALUES
     , ( 0x10dea67478c5f8c5e2d90e5e9b26dbe60c54d800, 'TAIKO', 18)
     , ( 0x80ac24aA929eaF5013f6436cdA2a7ba190f5Cc0b, 'syrupUSDC', 6)    
     , ( 0xa8258deE2a677874a48F5320670A869D74f0cbC1, 'WAR', 18)    
+    , ( 0xbeef69ac7870777598a04b2bd4771c71212e6abc, 'steakLRT', 18)
+    , ( 0x84631c0d0081fde56deb72f6de77abbbf6a9f93a, 'Re7LRT', 18)
+    , ( 0x7a4effd87c2f3c55ca251080b1343b605f327e3a, 'rstETH', 18)
+    , ( 0x5fd13359ba15a84b76f7f87568309040176167cd, 'amprETH', 18)
```
Comment (viniabussafi) [inline tokens/models/tokens/ethereum/tokens_ethereum_erc20.sql:4572]: https://etherscan.io/address/0x5fd13359ba15a84b76f7f87568309040176167cd

**Review by Hosuke — APPROVED** [review]
LGTM.✅
Thank you @viniabussafi

**Pull request merged**